src/Sulu/Bundle/SecurityBundle/Entity/TwoFactor/EmailInterface.php

```diff
@@ -17,12 +17,10 @@
  * Bridge interface to the scheb/2fa-email TwoFactorInterface.
  */
 if (\interface_exists(TwoFactorInterface::class)) {
-    /**
+    /*
      * @internal
      */
-    interface EmailInterface extends TwoFactorInterface
-    {
-    }
+    \class_alias(TwoFactorInterface::class, EmailInterface::class);
```

Comment:
need to be an alias instead of extending to not break `Contravariance` in `AuthCodeMailer`.